app/Http/Controllers/Admin/AdminController.php

```diff
@@ -49,11 +53,11 @@ public function store(AdminRequest $request)
 
             return redirect()
                 ->route('admin.admins.index')
-                ->with(['messages' => ['success' => ['Admin created Successfully']]]);
+                ->with(['messages' => json_encode(['success' => ['Admin created Successfully']])]);
```

Comment:
from now on use json_encode() like so, so you don't have to get messages and send it back to index, it is more robust now